classes/Product.php

```diff
@@ -6090,11 +6090,21 @@ public static function getAllCustomizedDatas($id_cart, $id_lang = null, $only_in
     }
 
     /**
+     * @deprecated since 8.0.0, the customization price impact is already included in Product::getPriceStatic.
+     *
      * @param array $products
      * @param array $customized_datas
      */
     public static function addCustomizationPrice(&$products, &$customized_datas)
     {
+        @trigger_error(
+            sprintf(
+                '%s is deprecated since version 8.0.0. The customization price impact is already included in Product::getPriceStatic.',
```

Comment:
```suggestion
                '%s is deprecated since version 9.0.0. The customization price impact is already included in Product::getPriceStatic.',
```